tasks/config/karma.js

```diff
@@ -49,6 +53,23 @@ module.exports = function (grunt) {
           { type: 'text-summary' },
         ]
       }
-    }
+    },
   };
+
+  times(TOTAL_CI_SHARDS, i => {
+    const n = i + 1;
+    config[`ciShard-${n}`] = {
+      singleRun: true,
+      options: {
+        files: [
+          'http://localhost:5610/bundles/commons.bundle.js',
+          `http://localhost:5610/bundles/tests.bundle.js?shards=${TOTAL_CI_SHARDS}&shard_num=${n}`,
+          'http://localhost:5610/bundles/commons.style.css',
+          'http://localhost:5610/bundles/tests.style.css'
+        ]
+      }
+    };
+  });
+
+  return config;
```

Comment:
Somewhat over-the-top comment added above: https://github.com/spalger/kibana/blob/3999f9f1fae951e5f39fa9de6cda2e4b70e86564/tasks/config/karma.js#L59